core/src/main/java/com/xatkit/core/recognition/nlpjs/NlpjsHelper.java

```diff
@@ -0,0 +1,126 @@
+package com.xatkit.core.recognition.nlpjs;
```

Comment:
Check that all if/else constructs are using curly braces.
This class needs documentation: it's not clear where it's used, why we need it, and what it does.

Reply:
This class provides heuristics to help adapt Xatkit parameter types to NLP.js style (ex: `var1:number` and `va2: number` -> `number_1` and `number_2`) and vice versa (ex: `number_1` and `number_2` -> `var1:number` and `va2: number`). We need a fix for the NLP.js bug to be able to use this mechanism though (I tested with NLP.js v3)

Reply:
I'll check the if/else constructs